src/quacc/schemas/ase.py

```diff
@@ -271,6 +271,40 @@ def summarize_vib_and_thermo(
     )
 
 
+def summarize_neb_run(dyn: Optimizer) -> OptSchema:
+    """
+    Summarizes the NEB run results into an OptSchema.
```

Comment:
What does this function need to do differently than `summarize_opt_run`. Is there a reason why we can't use that function or simply call that function and make modifications on the output?

Note: You aren't returning an `OptSchema` here. You have made up your own dictionary.